File: JAVA/pairSum.java
import java.util.*;

class HelloWorld {
    public static void main(String[] args) {
              int[] array = {1,2,3,4,5};
        int size = array.length;
        int targetSum = 5;
    pairSum(array, size, targetSum);
    }
    
    public static void pairSum(int[] arr, int size, int targetSum){

        for(int i = 0; i< size; i++){
            for(int j = i + 1; j < size; j++){
                if(arr[i] + arr[j] == targetSum){
                   System.out.println("(" + arr[i] + "," + arr[j] + ")" );
                }
            }
        }
    }
}

File: pairSum.js
function pairSum(array, targetSum) {
  for (let i = 0; i < array.length; i++) {
    for (let j = i + 1; j < array.length; j++) {
      if (array[i] + array[j] == targetSum) {
        console.log("(" + array[i] + "," + array[j] + ")");
      }
    }
  }
}

pairSum([1, 2, 3, 4, 5], 5);
